index.js

```diff
@@ -1,7 +1,57 @@
 const express = require('express'); 
 const app = express();
+const server = require('http').Server(app);
+const io = require('socket.io')(server);
+
+const players = {};
+
+io.on('connection', (socket) => {
```

Comment:
jak masz jeden argument to możesz pisać
socket =>
zamiast (socket) =>

Reply:
Albo `_` jak nie chcesz argumentu! XD

Reply:
Co to za szkoła :D wtedy jako argument masz `_`.
Jak chcesz bez argumentu to
`() => {}`

Reply:
Szkoła od zera do zerowego programisty JS! xD

Reply:
Oj tam oj tam :) dajesz radę!